### DISCUSSION TITLE
Can't kimai install and run as a built-in pph-S server on Windows 11 OS?

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#5031] [Plugins]
Hello,

Can't kimai install and run as a built-in pph-S server on Windows 11 OS?

php -c D:\kimai\php.ini D:\composer\composer.phar install --optimize-autoloader

When you execute the above command, an error occurs as shown below.

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 20480 bytes) in D:\kimai\vendor\symfony\error-handler\ErrorEnhancer\UndefinedFunctionErrorEnhancer.php on line 82

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 32768 bytes) in D:\kimai\vendor\symfony\error-handler\Error\OutOfMemoryError.php on line 1

D:\kimai\php.ini
memory_limit = 128

I increased memory_limit to 1024, 2048... or set it to -1 indefinitely, but the memory error continues to occur.

Can't I install kimai on Windows 11?

Also, is it possible to log in google saml as a localhost?

**Comment by seleondevel** [comment]
php bin/console about

If you also run the symfony version check command, you will get a memory error as shown below.

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 36864 bytes) in D:\kimai\vendor\symfony\error-handler\ErrorEnhancer\UndefinedFunctionErrorEnhancer.php on line 57

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 20480 bytes) in D:\kimai\vendor\symfony\error-handler\ErrorEnhancer\UndefinedFunctionErrorEnhancer.php on line 57

**Reply by kevinpapst** [reply]
Not a Kimai issue. Fix your PHP installation. 
https://www.php.net/manual/en/ini.core.php#ini.memory-limit
`memory_limit = 128M`
